test/test_pynisher.py

```diff
@@ -117,10 +118,16 @@ def crash_unexpectedly(signum):
 
 
 def crash_with_exception(exception):
-    print("going to raise {}.".format(exception))
+    print("going to raise {}.".format(exception.__name__))
     raise exception
 
 
+def keyboard_interruption():
+    time.sleep(0.5)
+    crash_with_exception(KeyboardInterrupt)
+    time.sleep(0.5)
```

Comment:
Could you please explain these two sleeps?

Reply:
I forgot removing those sleeps, so I will remove them.